### ISSUE TITLE
Update IDL

### ISSUE THREAD

**Issue by mitza-oci** [omg-dds/dds-security#19]
Would it work for everyone if we changed the contents of the IDL file in this repo to match https://github.com/omg-dds/dds-rtps/blob/master/srcCxx/shape.idl ?  It might require a PR for each implementation to adjust their build settings / configuration -- or it might just work out of the box.

@colmenero @JaimeMartin @ClarkTucker @ornulfs

**Comment by colmenero** [comment]
I'm ok with changing the contents of the IDL file to match the one in the [dds-rtps](https://github.com/omg-dds/dds-rtps/blob/master/srcCxx/shape.idl) repository.

To avoid breaking the state of the repository, how about we create an "epic" branch with the idl modification and all vendors test/implement based on that branch? We can merge it to master when ready.

**Comment by mitza-oci** [comment]
Yes, that sounds good.  I'll start a new branch after #16 and #17 are merged so we're at the same starting point.

**Comment by ClarkTucker** [comment]
Sounds good to me.
